<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testTaskComponents</class>
 <widget class="QWidget" name="testTaskComponents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="courseNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentScoreLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>15</y>
     <width>49</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxScoreLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>15</y>
     <width>49</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
